src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -119,14 +123,23 @@ public void parse_repeatedNonTagValue_failure() {
 
     }
 
-    //    @Test
-    //    public void parse_optionalFieldsMissing_success() {
-    //        // zero tags
-    //        Person expectedPerson = new PersonBuilder(AMY).withTags().build();
-    //        assertParseSuccess(parser, NAME_DESC_AMY + PHONE_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY
-    //                        + GENDER_DESC_AMY + AGE_DESC_AMY + DETAIL_DESC_AMY,
-    //                new AddCommand(expectedPerson));
-    //    }
+    @Test
+    public void parse_optionalFieldsMissing_success() {
```

Comment:
Is this test case same as `parse_allFieldsPresent_success()` on line 51?
The only optional field is Detail is included in this test, and omitting Detail test is in the next test method (`parse_detailFieldsMissing_success()`, line 136).

Reply:
This test case is testing the optional tag field and the case where no tags are present in the add command. I'll rename it to parse_tagFieldsMissing_success for clarity.

Reply:
Ah yes, I forgot about the Tags field. Yup, you are right on this.